src/overlays/actors/ovl_Boss_Va/z_boss_va.c

```diff
@@ -173,157 +294,3728 @@ static ColliderQuadInit D_8095BFF4 = {
+    if ((sCsCamera != 0) && (sCsState <= INTRO_TITLE)) {
+        Math_SmoothStepToF(&sCameraEye.x, sCameraNextEye.x, 0.3f, sCameraEyeMaxVel.x, 0.075f);
+        Math_SmoothStepToF(&sCameraEye.y, sCameraNextEye.y, 0.3f, sCameraEyeMaxVel.y, 0.075f);
+        Math_SmoothStepToF(&sCameraEye.z, sCameraNextEye.z, 0.3f, sCameraEyeMaxVel.z, 0.075f);
+        Math_SmoothStepToF(&sCameraAt.x, sCameraNextAt.x, 0.3f, sCameraAtMaxVel.x, 0.075f);
+        Math_SmoothStepToF(&sCameraAt.y, sCameraNextAt.y, 0.3f, sCameraAtMaxVel.y, 0.075f);
+        Math_SmoothStepToF(&sCameraAt.z, sCameraNextAt.z, 0.3f, sCameraAtMaxVel.z, 0.075f);
+        Gameplay_CameraSetAtEye(globalCtx, sCsCamera, &sCameraAt, &sCameraEye);
+    }
+}
+
+void BossVa_SetupBodyPhase1(BossVa* this) {
+    f32 endFrame = Animation_GetLastFrame(&gBarinadeBodyAnim);
+
+    Animation_Change(&this->skelAnime, &gBarinadeBodyAnim, 1.0f, endFrame, endFrame, ANIMMODE_ONCE, 0.0f);
+    this->actor.shape.yOffset = -450.0f;
+    this->actor.flags &= ~1;
+    this->timer = 25;
+    sBodyState = 0x80;
+    BossVa_SetupAction(this, BossVa_BodyPhase1);
+}
+
+void BossVa_BodyPhase1(BossVa* this, GlobalContext* globalCtx) {
+    Player* player = PLAYER;
+
+    this->unk_1B0 += 0xCE4;
+    this->bodyGlow = (s16)(Math_SinS(this->unk_1B0) * 50.0f) + 150;
+    if (this->timer != 0) {
+        this->timer--;
+        if (this->timer == 0) {
+            sBodyState &= (u8)~0x80;
+        }
+    }
+
+    if (this->colliderBody.base.atFlags & AT_HIT) {
+        this->colliderBody.base.atFlags &= ~AT_HIT;
+        if (this->colliderBody.base.at == &player->actor) {
+            func_8002F71C(globalCtx, &this->actor, 8.0f, this->actor.yawTowardsPlayer, 8.0f);
+        }
+    }
+
+    if (sBodyState & 0x7F) {
+        this->skelAnime.curFrame = 0.0f;
+        Actor_SetColorFilter(&this->actor, 0, 255, 0, 12);
+        Audio_PlayActorSound2(&this->actor, NA_SE_EN_BALINADE_DAMAGE);
+    }
+
+    if (SkelAnime_Update(&this->skelAnime) && (sFightPhase >= PHASE_2)) {
+        BossVa_SetupBodyPhase2(this, globalCtx);
+    }
+
+    Math_SmoothStepToS(&this->actor.shape.rot.x, this->actor.world.rot.x, 1, 0xC8, 0);
+    Math_SmoothStepToS(&this->actor.shape.rot.z, this->actor.world.rot.z, 1, 0xC8, 0);
+    this->unk_1AC += 0xC31;
+    this->unk_1A0 = (Math_CosS(this->unk_1AC) * 0.1f) + 1.0f;
+    this->unk_1A4 = (Math_SinS(this->unk_1AC) * 0.05f) + 1.0f;
+    if ((globalCtx->gameplayFrames % 4) == 0) {
+        BossVa_Spark(globalCtx, this, 1, 100, 50.0f, 10.0f, SPARK_BARI, 10.0f, false);
+    }
+
+    if (Rand_ZeroOne() < 0.1f) {
+        Audio_PlayActorSound2(&this->actor, NA_SE_EN_BALINADE_BL_SPARK - SFX_FLAG);
+    }
+
+    Collider_UpdateCylinder(&this->actor, &this->colliderBody);
+    CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->colliderBody.base);
+    CollisionCheck_SetAT(globalCtx, &globalCtx->colChkCtx, &this->colliderBody.base);
+    func_800F436C(&this->actor.projectedPos, 0x313C, 1.0f);
```

Comment:
might be some more in the file, run the script to catch them (if it is set up to recognize this function at least)